### ISSUE TITLE
Feature suggestion: option to ignore service/website

### ISSUE THREAD

**Issue by joubertredrat** [phpservermon/phpservermon#319]
Hi guys,

I think that is good idea to have option to ignore service/website to verifying. Lets to example :)

You have one site and you need to change DNS entries, between one day this site will down, then, Will not cool phpservermon check this site if you know that is offline, then you can set this site to ignore and when you finish your maintenance, you can remove ignore.

What you think guys?

**Comment by sadortun** [comment]
Good idea, If you are willing to submit a PR, we can add the feature in the next release.

**Comment by joubertredrat** [comment]
Hi @sadortun 

I'm looking here on code and have a similar function, named "monitoring", but on my idea, on view is different. If you disable monitoring, is display 'ignore' on servers list and servers status, as example below.

![screenshot-uptime vectornet etc br 2016-06-21 09-18-59](https://cloud.githubusercontent.com/assets/1520407/16229286/59a7346e-3792-11e6-9ba2-e902f3d4d34f.png)
![screenshot-uptime vectornet etc br 2016-06-21 09-22-47](https://cloud.githubusercontent.com/assets/1520407/16229292/5e8af290-3792-11e6-83b0-ac864b01b148.png)
![screenshot-uptime vectornet etc br 2016-06-21 09-25-52](https://cloud.githubusercontent.com/assets/1520407/16229295/6201d2ae-3792-11e6-9bbe-6575ac3f2602.png)

If you like this, talk me that I change this here.

**Comment by sadortun** [comment]
Hi @joubertredrat Sounds good!

@dopeh What do you think about this ? Is it what you had in mind by Monitoring ?

**Comment by joubertredrat** [comment]
ping @sadortun @dopeh

**Comment by sgregori** [comment]
Any news about this feature ? 

Will be lovely ;)

ping @joubertredrat @sadortun @dopeh
